### PR TITLE
AWS Lambda - Update Layer Version Command Enhancement

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
The aws-lambda-publish-layer-version now supports the 'compatible-architectures' argument.

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py [hunk 1]
```diff
@@ -702,7 +703,7 @@ def list_layer_version_command(args, aws_client):
         'CompatibleRuntime': args.get('compatible-runtime'),
         'Marker': args.get('token'),
         'MaxItems': arg_to_number(args.get('limit')),
-        'CompatibleArchitecture': args.get('compatible-architecture')
+        'CompatibleArchitecture': args.get('compatible_architecture')
```
Comment (RotemAmit) [inline Packs/AWS-Lambda/Integrations/AWS_Lambda/AWS_Lambda.py:706]: Please revert the change.
```suggestion
        'CompatibleArchitecture': args.get('compatible-architecture')
```

**Review by RotemAmit — COMMENTED** [review]
Nice work!

**Review by altmannyarden — APPROVED** [review]
